.github/workflows/publish-electricity-market-integration-bundle.yml

```
@@ -0,0 +1,94 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Publish ElectricityMarket integration bundle
+
+on:
+  # Build, test, pack and publish release packages
+  # Only executed if package content; not if solution file, documentation or tests has changed
+  push:
+    branches:
+      - main
+    paths:
+      - source/Energinet.DataHub.ElectricityMarket.Integration/**
+      - .github/workflows/publish-electricity-market-integration-bundle.yml
+  # Build, test, pack
+  # Executed if package contenet has changed
+  workflow_call: {}
+  # Build, test, pack and publish prerelease packages
+  # Executed if manually triggered
+  workflow_dispatch: {}
+
+env:
+  # Conditions
+  PUSH_PACKAGES: ${{ true }}
```

I ønsker vel kun at publisere pakker når PR er merged? (samme adfærd som geh-core)
```suggestion
  PUSH_PACKAGES: ${{ github.event_name != 'pull_request' }}
```